ring-core/test/ring/core/test/protocols.clj

```diff
@@ -53,3 +54,36 @@
           response {:body nil}]
       (write-body-to-stream (:body response) response output)
       (is (= "" (.toString output))))))
+
+(defn output-stream-with-close-flag []
+  (let [is-stream-closed (atom false)
```

Comment:
`is-stream-closed` should be `stream-closed?` to fit in with Clojure naming schemes.